src/cli/commands/deploy.ts

```diff
@@ -16,12 +16,12 @@ export async function deploy(network: string, contractName: string, contractArgs
     ...saddle.network_config.defaultOptions,
     from: saddle.account
   };
-  let {contract, receipt} = await deployContract(saddle.web3, network, contractName, contractArgs, trace, saddle.network_config.defaultOptions, sendOptions);
+  let {contract, receipt} = await deployContract(saddle.web3, network, contractName, contractArgs, saddle.network_config.build_dir, trace, saddle.network_config.defaultOptions, sendOptions);
```

Comment:
OK, I can get down with just doing it in imports. The spaces vscode puts in these declarations gives me chills